ern-core/src/android.ts

```diff
@@ -81,6 +83,20 @@ async function delay(ms: number) {
   })
 }
 
+// Returns a promise that will get rejected after a given timeout (in ms)
+async function failAfterTimeOut(ms: number) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(
+        new Error(
+          `Failed to collect the device/emulator state
+        ${ANDROID_DEVICE_INFO}`
```

Comment:
Because the new `ANDROID_DEVICE_INFO` constant is a template string that _begins_ with `LF`, this error will now have 8 trailing spaces on the second line (and nothing else). Similar issue on line 209 below.
Can you either remove the remove the linefeed from the template string, or (maybe better) remove the indentation if this variable here?

Reply:
@friederbluemle here's the error 
```
✖ An error occurred: Failed to collect the device/emulator state
✖
✖ https://developer.android.com/studio/run/emulator-commandline
✖ https://developer.android.com/studio/run/emulator
```